src/06-sort.js

```diff
@@ -33,7 +46,14 @@ function sortByArtistNameDescending(songs) {}
  * @param {Object[]} songs - An array of songs. See the song data for more.
  * @returns {Object[]}
  */
-function sortBySongTitleAscending(songs) {}
+
+
+
+ // -------------  sorts song title in ascending - one liner - uses sort & Arrow function -----------------
+ // Here we use .LocaleCompare() which compares ASCII value before sorting. - localeCompare is mainly used for non english characters
+ // Used an alternative for previous function to show variation - both method works
+
+ const sortBySongTitleAscending = (songs) => songs.sort((a, b) => a.title.localeCompare(b.title))
```

Comment:
That's one way to do it! However, it is best to use the approach you used on `sortArtistNameDescending` as it is more inline with how sort behaves